config/application.rb

```diff
@@ -63,7 +63,7 @@ class Application < Rails::Application
       # registry config needed
       config.autoloader = :classic
       config.registry_private_key = OpenSSL::PKey::RSA.new(
-        File.read('/etc/rmt/ssl/rmt-server.key')
```

Comment:
Can this be moved from general RMT code into the registry engine that uses this?

Reply:
Done in #1155